examples/shopify/README.md

```diff
@@ -30,66 +30,40 @@ NEXT_PUBLIC_SHOPIFY_ACCESS_TOKEN=b434d672242174f77e306910462c3d67
 
 ## Using this repo
 
-1. **Create a site in Makeswift and a shop in Shopify**
+> **Note**
```

Comment:
I will say that I would like to keep this one in sync with the bigcommerce readme as much as possible. It's just makes updates easier. The more they all diverge the lower the quality will go i think. 

I do like the bigcommerce format more, but it can be your call